### ISSUE TITLE
Picker Accessibility: Does it announce "disabled" or Checked/Unchecked State

### ISSUE THREAD

**Issue by amarlette** [react-native-picker/picker#215]
# Slider Accessibility
In regards to facebook/react-native#30940 & facebook/react-native#30962 has react-native-picker already addressed these accessibility issues? Any help in identifying if this issue has been addressed in this repo would be greatly appreciated!

### Expected results
When a component is "disabled", screen readers should announce this as part of their focus announcement. Clicks to these elements should also have no effect.

### Expected results
Upon focus of an element with the "checked" accessibilityState, the screen reader should announce either "checked" or "not checked", or in the case of elements with the "switch" role, it should announce "on" or "off". Upon activation, if the state is changed, the new state should be announced.
